definitions/npm/match-sorter_v5.x.x/flow_v0.104.x-/match-sorter_v5.x.x.js

```diff
@@ -0,0 +1,35 @@
+type $npm$matchSorter$KeyFn<T> = (item: T) => string | Array<String>;
+type $npm$matchSorter$KeyObj = {|
+  key: string,
+  minRanking?: number,
+  maxRanking?: number,
+  threshold?: number,
+|};
+
+type $npm$matchSorter$Rankings = {|
```

Comment:
Is there any benefit to keep those type globally scoped as is, that used to be the "default" in old lidbefs but should be questioned IMO.

Reply:
Nope; I'm pretty new to writing libdefs, so I wasn't aware of the current best practices. I'll change these to not be globally scoped.

Reply:
I updated the code to not use global types.

I wasn't sure whether to export these intermediate types, so I decided not to initially. Let me know if you disagree.

Reply:
It's fine, It'll be just a PR way if someone needs them.